tt_metal/hw/firmware/src/erisc.cc

```diff
@@ -89,12 +89,14 @@ void __attribute__((section("erisc_l1_code"))) ApplicationHandler(void) {
     for (uint32_t n = 0; n < NUM_NOCS; n++) {
         noc_local_state_init(n);
     }
+    DEBUG_STATUS('I');
```

Comment:
Just added a couple sample DEBUG_STATUSes here, we can modify/expand these as we see what parts are important for debugging moving forward.